application-workloads/wordpress/wordpress-app-service-mysql-inapp/azuredeploy.json

```diff
@@ -59,7 +59,7 @@
       "location": "[parameters('location')]",
       "properties": {
         "name": "[variables('siteName')]",
-        "serverFarmId": "[variables('hostingPlanName')]",
+        "serverFarmId": "[reference(resourceId('Microsoft.Web/serverFarms/', variables('hostingPlanName'))).name]",
```

Comment:
Just use:
```resourceId('Microsoft.Web/serverFarms', variables('hostingPlanName'))```

Reply:
Hi @bmoore-msft  previously set value was the name of the resource. I can add the resource id which should be fine. but semantically it may not be correct. What do you think?

Reply:
If it doesn't accept the resourceId, then we need to fix that in the TTK... so try that and let's check the deployment.

Reply:
Thanks @bmoore-msft, resource Id accepted.